doc/man7/flux-environment.rst

```diff
@@ -170,8 +179,10 @@ environment in other workload managers, for example:
    BATCH_NCORES=$(flux resource list -n -o {ncores})
    BATCH_NGPUS=$(flux resource list -n -o {ngpus})
    BATCH_HOSTLIST=$(flux getattr hostlist)
-   BATCH_JOBID=$(flux getattr jobid)
 
+Additionally, :envvar:`FLUX_ENCLOSING_ID` is set by the broker if it was
+launched as part of a Flux job, and contains the jobid of the instance
+in F58 form.
```

Comment:
maybe just "launched as a Flux job", not sure "part of" is necessary

Reply:
Well, a single broker is not "a Flux job" unless the job has only one node.. but either way gets the point across if you feel strongly about it.

Reply:
ohhh i see your context on it now.  Perhaps just "is set", no need to say "set by the broker"?  Dunno if the latter is a detail the user needs to know.

Reply:
Good point, that will probably make it clearer to the intended audience.

Reply:
Simplified to duplicate @garlick's text from above:
> Additionally, :envvar:`FLUX_ENCLOSING_ID` is set to the jobid of the enclosing instance, if it has one.

How's that?

Reply:
yup i think that works.  I guess you could add back the `F58 form`, but I don' think that matters.